### DISCUSSION TITLE
Entità con valore zero - Tasmota, MQTT, pzem016

### DISCUSSION THREAD

**Discussion by nicolacampagnolo** [indomus/forum#356] [GENERALE - Discussioni generiche]
Ciao a tutti,
tempo fa avevo integrato un sensore pzem016 e relativo wemosD1 seguendo questa [guida](https://indomus.it/guide/integrare-le-letture-di-assorbimento-elettrico-di-pzem-su-home-assistant-via-mqtt/) e tutto funziona corettamente.
Oggi ho voluto aggiungere altri due sensori pzem016b e relativi wemosD1, e dopo aver rifatto tutto come da guida seguendo passo passo le varie configurazioni mi ritrovo con il seguente problema.

In pratica via MQTT le letture vengono inviate e ricevute correttamente su HA quando mi metto in ascolto, allego screenshot (questo è uno dei due sensori appena aggiunti):

![Screenshot 2021-05-24 21 09 18](https://user-images.githubusercontent.com/74872434/119396142-c63e6a80-bcd4-11eb-998b-49ebfa70bcdf.png)

Ma poi se vado a vedere le varie entità sulla plancia vedo questo. Ovvero il primo sensore installato che funziona perfettamente, mentre gli altri due hanno valore zero. Questo vale per tutte le varie entità aggiunte oggi. Allego screenshot.

![Screenshot 2021-05-24 21 06 55](https://user-images.githubusercontent.com/74872434/119396317-fede4400-bcd4-11eb-844c-bcf68e28224d.png)

Ho provato a riavviare varie volte tutti i dispositivi, ma senza esito.
Ho controllato le configurazioni, ma per quel che ne capisco mi pare tuto ok.

Qualche suggerimento?

Grazie e buona serata.

**Comment by snakuzzo** [comment]
Se via MQTT il valore è inviato correttamente e su HA vedi 0, probabilmente hai sbagliato a scrivere il template degli altri due sensori

**Reply by nicolacampagnolo** [reply]
Risolto.
Era un errore in fase di configurazione della entità.

Grazie.
Nicola.
